File: CPP/Day10.cpp
/*

Program to accept the height of a person in cm and convert it to feet. 

*/

//Program
#include <iostream>
using namespace std;

int main() {
   float height_cm, height_ft;

   cout << "Enter height in centimeters: ";
   cin >> height_cm;

   height_ft = height_cm / 30.48;

   cout << "Height in feet: " << height_ft << endl;

   return 0;
}


/*

Input and Output:
Enter height in centimeters: 180
Height in feet: 5.90551

*/

File: Java/Day10.java
/*

Day 10: Accept the height of a person in cm and convert it to feet.

*/

// Program
import java.util.Scanner;

public class Day10{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the height in cm: ");
        int h = sc.nextInt();
        System.out.println("Height in feet is "+ h*0.0328084);
    }
}

/*

Input/Output:

Enter the height in cm: 
190
Height in feet is 6.233596

*/
